web/src/config.ts

```diff
@@ -0,0 +1 @@
+export const approvedAgreementRequiredReadingSeconds = 90;
```

Comment:
thoughts on naming and what this number should be are very welcome

Reply:
>That would be really nice – any ideas how within the constraints of uikit?

maybe something like (assuming we add `className={styles["agreement-button"]}`
```css
.agreement-button {
transition: background-color 90s ease;
background-color: #0d68cf;

&:disabled {
 background-color: #808080; or var(--grey)
}
}
```

Reply:
won't allow us to do anything when it's actually disabled, so we could set the disabled state for 60-80s and the fade in will happen after that, which can take 30-40s and will give the illusion of being disabled for 90-100s

```css
.agreement-button {
  width: 100%;
  transition: background-color 30s ease;
  background-color: #0d68cf;
}
```

Reply:
@jhughes982 what do you think?